### PR TITLE
adds mission comment to footer

### PR DESCRIPTION
**Pull request by jtmst**
## Description
Relates to #[14875](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14875)

## Testing done
Local visual testing

## Screenshots
Screen Shot 2023-09-12 at 10 09 55 AM

## QA steps

```[tasklist]
- [ ] Automated tests have passed
- [ ] Tugboat environment was generated without errors
- [ ] Verify comment is in footer
```

### CODE REVIEW

**Review by tjheffner — COMMENTED** [review]
I like this approach, but I have two things.

1. i'd either include this directly in the `footer/index.tsx` file (a separte const above the exported footer const should be fine)

or 

2. make the hook generic by accepting a comment string & an element to be passed to the `document.querySelector`. 

if you go with option 2, the hook could be re-used for the development info comment in the head. it could either live in utils or in src/hooks, depending on how it's structured

File: src/templates/globals/utils/FooterComment.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { useEffect } from 'react'
+
+const FooterComment = () => {
```
Comment (timcosgrove) [inline src/templates/globals/utils/FooterComment.tsx:3]: Is it possible to have a general `Comment` component and pass in the comment as a child prop?

Comment (jtmst) [reply src/templates/globals/utils/FooterComment.tsx:3]: good call as I'll be using this kind of comment in the head as well, will refactor

**Comment by timcosgrove** [comment]
We will not be doing this.

**Pull request closed without merging**